pkg/api/vm/formatter.go

```diff
@@ -462,3 +474,17 @@ func canCPUAndMemoryHotplug(vm *kubevirtv1.VirtualMachine) bool {
 	}
 	return !hasRestartRequiredOrHotplugMigration
 }
+
+func canHotplugNic(vm *kubevirtv1.VirtualMachine) bool {
+	if vm.Status.PrintableStatus != kubevirtv1.VirtualMachineStatusRunning {
```

Comment:
Is it duplicated with the check `canMigrate->ValidateVMMigratable` ?


https://github.com/harvester/harvester/blob/f16f02037000971c3a4fa3bd46a6c80593e183ae/pkg/util/virtualmachineinstance/virtualmachineinstance.go#L73-L75

Reply:
addressed in https://github.com/harvester/harvester/pull/9395/commits/b2e43a9cd9b7c565e39442a9937e5127037761d2